packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -170,18 +184,23 @@ const initState = (
   // In the event they're needed they can be reconstructed from contract terms and off-chain data.
   const vaultParamManagers = makeScalarMap('vaultParamManagers');
 
-  const { metricsPublication, metricsSubscription } = makeMetricsPublisherKit(
-    storageNode,
+  const { publisher: metricsPublisher, subscriber: metricsSubscriber } =
+    makeVaultDirectorMetricsPublishKit();
+
+  const storedMetricsSubscriber = makeStoredSubscriber(
+    metricsSubscriber,
+    E(storageNode).makeChildNode('metrics'),
     marshaller,
   );
 
+  const managerBaggages = provideChildBaggage(baggage, 'Vault Manager baggage');
```

Comment:
There's no call on `children()`, so these aren't getting revived.

Here's an example of [reviving from a set of baggages](https://github.com/Agoric/agoric-sdk/blob/8f9e16bfcb18921b9d4a2e10f198362513f711ad/packages/ERTP/test/swingsetTests/ertpService/vat-ertp-service.js#L34)

It may be time to use the new test framework from #6715 to test contract upgrade.

Reply:
Yeah, revival is explicitly out of scope for this PR. 

> On the path to https://github.com/Agoric/agoric-sdk/issues/5285 but doesn't claim to finish it.
>
> The goal of this is to use the new durable publish kits.